backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -270,12 +376,19 @@
                                       email)))))
 
   (users [this tenant-label]
-    (tenant-members this tenant-label)))
+    (tenant-members this tenant-label))
+
+
+  p/KeycloakAuthorization
+  (allowed-paths [keycloak email]
+    (allowed-paths keycloak email)))
 
-(defn- keycloak [{:keys [credentials url realm]}]
-  (map->KeycloakAgent {:issuer (format "%s/realms/%s" url realm)
-                       :api-root (format "%s/admin/realms/%s" url realm)
-                       :credentials credentials}))
+(defn- keycloak [{:keys [credentials http-pool url realm]}]
+  (map->KeycloakAgent {:api-root (format "%s/admin/realms/%s" url realm)
+                       :credentials credentials
+                       :issuer (format "%s/realms/%s" url realm)
+                       :user-id-cache (atom {})
```

Comment:
Using an atom (instead of a size-limited cache) because we do not expect too many users?

Reply:
Should change

Reply:
should be solved by https://github.com/akvo/akvo-lumen/pull/2227/commits/b1910251dd800db281fc98b27594b2640e1e6d41